### ISSUE TITLE
No Trades on modified Jobless Villagers

### ISSUE THREAD

**Issue by manusoftar** [Luke100000/minecraft-comes-alive#892]
**Describe the bug**
I don't know if I'm doing something wrong but whenever I approach a villager and open the interactions menu the Trade or Trading option never appears so I cannot trade... I'm using the mod on Minecraft Forge 1.20.1

**To Reproduce**
1. Get close to an MCA Villager
2. Right click to open the interactions menu

**Version**
MCA Version 7.5.8, Minecraft version 1.20.1, Modloader Forge

**Comment by Luke100000** [comment]
If the trade button does not appear they do not offer to trade (e.g. guards or jobless villagers)

**Comment by manusoftar** [comment]
Not necesarily, on one of the versions of One Block, jobless villagers
spawn (by data pack) and have trades setted. Also wandering villagers have
trades even though they don't have a job.

**Comment by Luke100000** [comment]
You should have mentioned earlier that you modded them. You can always bypass MCA by shift right clicking

**Comment by manusoftar** [comment]
I didn't mod anything, I'm using a map called Skyblock One Block which
makes use of a data pack for it's functionality, it is in no way a mod,
it's a map which starts with only one single dirt block that you break and
it respaws but it gives you blocks and items and it begins to change the
longer you mine it, but since it has to respawn and sometimes change it
requires a data pack for the commands that it has to run, it's too complex
for a command block or a collection of command blocks.

And wandering villagers are vanilla so no modding there neither.

**Comment by Luke100000** [comment]
Please link the pack!

> wandering villagers are vanilla

Do you mean wandering traders? Those are not villagers.

**Comment by manusoftar** [comment]
yes, sorry, my bad, I meant wandering traders, when I right click on a
wandering trader I get the usual MCA dialog but the trade option is not
enabled.
I attached the datapack used on the Oneblock map.
